chcemvediet/apps/accounts/tests/test_forms.py

```diff
@@ -74,54 +73,59 @@ def test_post_signup_form_with_invalid_data_does_not_create_user(self):
     def test_first_name_field_is_required(self):
         data = self._create_account_signup_data(first_name=u'')
         response = self.client.post(reverse(u'account_signup'), data, follow=True)
-        self.assertFormError(response, u'form', u'first_name', 'This field is required.')
+        self.assertFormError(response, u'form', u'first_name', u'This field is required.')
```

Comment:
@martinmacko47 Alebo mam nechat obycajne uvodzovky `''This field is required.''`, aby som poukazal nato, ze sa porovnava obycajny string `<type: str>`

Reply:
@viliambalaz Vsade by sme mali pouzivat unicode stringy. Cize ano, dobre, ze si to zmenil na `u'...'` stringy. Non-unicode stringy by sme mali pouzivat iba na miestach, kde sa nedaju.